fairlearn/postprocessing/_interpolated_thresholder.py

```diff
@@ -9,7 +9,8 @@
 from sklearn.utils.validation import check_is_fitted
 from warnings import warn
 
-from fairlearn._input_validation import _validate_and_reformat_input
+from ..utils._common import _get_soft_predictions
+from .._input_validation import _validate_and_reformat_input
```

Comment:
Out of curiosity: Is this the preferred notation? I always liked the explicit full path like `fairlearn._input_validation`, but I am not aware of any guidance either way.

Reply:
full path would import from your installed version, not this directory, this would import from this (or relative) directory. Some libraries do relative import in code, some do absolute.